.github/workflows/docker_image_build.yaml

```diff
@@ -1,9 +1,13 @@
 # Docker images are re-built only when the corrsponding docker files under Docker/ are changed
+# setting branches is to omit this action when tagging
+#
 name: Building the Docker images containing the basic tools for building repos
 on:
   push:
     paths: 
       - 'Docker/**'
+    branches: 
+      - '**'
```

Comment:
Didn't you just say you want this to trigger on a push to master? AFAIU this will trigger on a push to any branch.